packages/gasket-plugin-data/lib/index.d.ts

```diff
@@ -17,7 +16,8 @@ declare module '@gasket/core' {
   import { RequestLike, GasketRequest } from '@gasket/request';
 
   export interface GasketConfig {
-    data?: GasketDataDefinition
+    data?: GasketDataDefinition;
+    command?: string;
```

Comment:
Shouldn't `command` should be declared on GasketConfig in the commands plugin, not here in data plugin.